hamlpy/test/templates/filtersPygments16.hamlpy

```diff
@@ -1,7 +0,0 @@
-:highlight
```

Comment:
These files were previously only being tested if pygments 1.6 was installed... which is never if you have latest pygments loaded. I think it's ok to only test against latest pygments.